.github/workflows/crucible-go-build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
-        ghc: ["8.10.7", "9.0.2", "9.2.3"]
+        ghc: ["8.10.7", "9.0.2", "9.2.4"]
         include:
           - os: macos-12
             ghc: 9.2.2
```

Comment:
Shouldn't the OSes listed under the `include`s also be updated to GHC 9.2.4? (This is one of them, but there are others as well.)

Reply:
Good catch: I'd just looked for 9.2.3 but hadn't scanned the entire yaml file.  This is now updated.